keyboards/nuphy/air75_v2/ansi/user_kb.c

```diff
@@ -0,0 +1,419 @@
+/*
+Copyright 2023 @ Nuphy <https://nuphy.com/>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "user_kb.h"
+#include "ansi.h"
+#include "usb_main.h"
+#include "rf_driver.h"
+
+kb_config_t     kb_config;
+DEV_INFO_STRUCT dev_info = {
+    .rf_baterry = 100,
+    .link_mode  = LINK_USB,
+    .rf_state   = RF_IDLE,
+};
+bool f_bat_hold         = 0;
+bool f_sys_show         = 0;
+bool f_sleep_show       = 0;
+bool f_send_channel     = 0;
+bool f_dial_sw_init_ok  = 0;
+bool f_rf_sw_press      = 0;
+bool f_dev_reset_press  = 0;
+bool f_rgb_test_press   = 0;
+bool f_bat_num_show     = 0;
+
+uint8_t        rf_blink_cnt          = 0;
+uint8_t        rf_sw_temp            = 0;
+uint8_t        host_mode             = 0;
+uint16_t       rf_linking_time       = 0;
+uint16_t       rf_link_show_time     = 0;
+uint16_t       no_act_time           = 0;
+uint16_t       dev_reset_press_delay = 0;
+uint16_t       rf_sw_press_delay     = 0;
+uint16_t       rgb_test_press_delay  = 0;
+host_driver_t *m_host_driver         = 0;
+
+extern bool               f_rf_new_adv_ok;
+extern report_keyboard_t *keyboard_report;
+extern report_nkro_t *nkro_report;
+extern uint8_t            bitkb_report_buf[32];
+extern uint8_t            bytekb_report_buf[8];
+extern uint8_t            side_mode;
+extern uint8_t            side_light;
+extern uint8_t            side_speed;
+extern uint8_t            side_rgb;
+extern uint8_t            side_colour;
+
+/**
+ * @brief  gpio initial.
+ */
+void gpio_init(void) {
+    /* enable led power driver  */
+    setPinOutput(DRIVER_LED_CS_PIN);
+    setPinOutput(DRIVER_SIDE_CS_PIN);
+    writePinLow(DRIVER_LED_CS_PIN);
+    writePinLow(DRIVER_SIDE_CS_PIN);
+    /* set side led pin output low */
+    setPinOutput(DRIVER_SIDE_PIN);
+    writePinLow(DRIVER_SIDE_PIN);
+    /* config RF module pin */
+    setPinOutput(NRF_WAKEUP_PIN);
+    writePinHigh(NRF_WAKEUP_PIN);
+    setPinInputHigh(NRF_TEST_PIN);
+    /* reset RF module */
+    setPinOutput(NRF_RESET_PIN);
+    writePinLow(NRF_RESET_PIN);
+    wait_ms(50);
+    writePinHigh(NRF_RESET_PIN);
+    /* config dial switch pin */
+    setPinInputHigh(DEV_MODE_PIN);
+    setPinInputHigh(SYS_MODE_PIN);
+    /* open led DC driver */
```

Comment:
Capitalize acronym:

```suggestion
    /* open LED DC driver */
```

Reply:
Thank you for your suggestions. I will carefully check the spelling and grammar.

Reply:
@nuphy-src I see you marked all 16 of my suggestions as “resolved”. However, I still see all of those mispellings and formatting errors in your latest changes (3995b3669c0bbabac2779f5a86dbbf1df4f6d130).

Please un-resolve my suggestions so that other reviewers can see that they have not been addressed yet. Even if you don’t intend to fix those issues before getting the PR merged (though I hope you do), leaving the comments un-resolved will save the time of reviewers [like @pdf](https://github.com/qmk/qmk_firmware/pull/22751#discussion_r1639629674) who notice and write about the same problems.